lib/rys/engine_extensions.rb

```diff
@@ -124,6 +124,13 @@ def rys_id(name=nil)
         @rys_id ||= ActiveSupport::Inflector.underscore(self.name).sub('/engine', '')
       end
 
+      def plugin_active?
+        true
+      end
+
+      def deactivated_plugin_html(_view_context)
```

Comment:
Prosím k čemu je tato metoda? Bylo by možné přidat komentář který by vysvětloval její použití? Za rok nikdo nebude vědět na co to je... 
díky